app/controllers/problems_controller.rb

```diff
@@ -250,6 +250,14 @@ def destroy
     end
   end
 
+  # Remove the current problem from the selected problem set and refresh the page
+  def remove_from_problem_set
+    @problem_set = ProblemSet.find(params[:problem_set_id])
```

Comment:
You are right. I was copying from a similar function under the problem set controller because I don't know much about ruby either. Should I change it?

Reply:
Just checking, you mean places like [here](https://github.com/puqeko/nztrain/blob/022266ef82710c3788e0d0fe578b3cf1e16308c3/app/controllers/problem_sets_controller.rb#L12)? I'm not sure, but perhaps that is an instance variable because it's used in the template when the page is rendered? I can do some more research tomorrow.

Reply:
Yes, I pretty much copy-pasted the `remove_problem` method just below it. Your presumption seems logical. I should also be able to have a look tomorrow afternoon. Good night :)

Reply:
04d5ce6a8aabd58b01fd504b6a9e586c2b4c73db removes the unnecessary instance variable, and seems to work. I'll try deploying shortly.